packages/mdc-chips/chip-set/adapter.js

```diff
@@ -15,7 +15,8 @@
  * limitations under the License.
  */
 
-/* eslint no-unused-vars: [2, {"args": "none"}] */
+/* eslint-disable no-unused-vars */
+import {MDCChipInteractionEventType} from '../chip/foundation';
```

Comment:
I think we want to keep the `no-unused-vars: [2, {"args": "none"}]` rule for the rest of the file, so you can do something like this:
```
// eslint-disable-next-line no-unused-vars
import {MDCChipInteractionEventType} from '../chip/foundation';

/* eslint no-unused-vars: [2, {"args": "none"}] */
...
```

Reply:
Cool, done!